iree/modules/check/check_module_main.cc

```diff
@@ -181,8 +186,16 @@ StatusOr<int> Run() {
 
 extern "C" int main(int argc, char** argv) {
   InitializeEnvironment(&argc, &argv);
+  IREE_FORCE_BINARY_STDIN();
+
+  if (argc < 2) {
+    LOG(ERROR)
+        << "A binary module file path to run (or - for stdin) must be passed";
+    return -1;
+  }
+  auto input_file_path = std::string(argv[1]);
```

Comment:
Why is changing this to a positional argument necessary? I think whatever changes we make here should also be reflected in iree-run-module and iree-benchmark-module, which currently use the same format (can be a followup). I guess that will be necessary to make them run on Windows anyway?

Reply:
I have a bug tracking the same work for iree-run-mlir (#1281). I don't want proliferation of the bad style as you start to ramp up tests on this - easier to fix now.

Reply:
Ok. Still seems like flag to positional argument is unrelated, but I'm fine with either as long as it's consistent between tools.